src/main/java/utils/ConfigReader.java

```diff
@@ -1,19 +1,55 @@
 package utils;
 
 import java.io.FileInputStream;
-import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.util.Properties;
 
 public class ConfigReader {
-    private Properties properties;
 
-    public ConfigReader(String configFileName) throws IOException {
-        properties = new Properties();
-        properties.load(new FileInputStream("src/main/resources/config/" + configFileName));
+    private Properties commandNameToClass = new Properties();
+    private Properties appConfig = new Properties();
+    private Properties arangoConfig = new Properties();
+
+    private volatile static ConfigReader instance;
+
+    public static boolean isTesting;
+
+    private ConfigReader() throws IOException {
+        populateWithConfig("commands.config", commandNameToClass);
+        populateWithConfig(isTesting ? "arango.test.config" : "arango.config", arangoConfig);
+        populateWithConfig("app.config", appConfig);
+    }
+
+    private static void populateWithConfig(String configFileName, Properties properties) throws IOException {
+        String configFolder = "src/main/resources/config/";
```

Comment:
I think we have to leave the full path of config file to the initializer of microservice. I know that the best practice is to add any configuration files in resource folder but I think we should make it fully dynamically specified by administrator starting the microservice. What do you think?